Library/Formula/polo.rb

```diff
@@ -0,0 +1,39 @@
+require "language/go"
+
+class Polo < Formula
+  desc "Static blog generator compatible with Jekyll & Pelican content"
+  homepage "https://github.com/agonzalezro/polo"
+  url "https://github.com/agonzalezro/polo/archive/v1.0.tar.gz"
+  sha256 "39ba03b17f725faf319c97a3e9e5852e4abb8c420e3cd4216b7c1a0ba7dcb7ff"
+
+  depends_on "go" => :build
+
+  go_resource "github.com/constabulary/gb" do
+    url "https://github.com/constabulary/gb.git",
+        :revision => "23f405f2981056a97abb52b5b7e528a6cddef1bb"
+  end
+
+  def install
+    ENV["GOPATH"] = buildpath
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    cd "src/github.com/constabulary/gb" do
+      system "go", "get", "./..."
```

Comment:
Probably yes. It will download the `gb` dependencies into the `$GOPATH`, in
this case `buildpath`.

I tried to find another example about how to do it on `homebrew` codebase
but I couldn't find anything.

On Wed, Nov 18, 2015 at 8:04 PM, Dominyk Tiller <notifications@github.com>
wrote:

> In Library/Formula/polo.rb
> <https://github.com/Homebrew/homebrew/pull/44796#discussion_r45243759>:
>
> > +  url "https://github.com/agonzalezro/polo/archive/v1.0.tar.gz"
> > +  sha256 "39ba03b17f725faf319c97a3e9e5852e4abb8c420e3cd4216b7c1a0ba7dcb7ff"
> > +
> > +  depends_on "go" => :build
> > +
> > +  go_resource "github.com/constabulary/gb" do
> > +    url "https://github.com/constabulary/gb.git",
> > +        :revision => "23f405f2981056a97abb52b5b7e528a6cddef1bb"
> > +  end
> > +
> > +  def install
> > +    ENV["GOPATH"] = buildpath
> > +    Language::Go.stage_deps resources, buildpath/"src"
> > +
> > +    cd "src/github.com/constabulary/gb" do
> > +      system "go", "get", "./..."
>
> So there's no remote fetching happening here at all?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/44796/files#r45243759>.
>



-- 
@agonzalezro <http://twitter.com/agonzalezro> or my blog
<http://agonzalezro.github.io>
Please, don't send me files with extensions: .doc, .docx, .xls, .xlsx, .ppt
and/or .pptx